### ISSUE TITLE
Named mob spawn quests

### ISSUE THREAD

**Issue by pandapaul** [ToroCraft/Dailies#44]
This should be an entirely new quest type so that it is handled differently in terms of events and such.  Spawn a mob with a name tag attached to it having some spiffy name based on the type of mob. e.g. A bat named Bruce.  

Need to decide spawn location, but off the top of my head I'd guess something like between a radius of 32 and 64 blocks away from the player.  Pick something that keeps the mob from spawning right on top of you but also in a reasonable distance away.  

Perhaps the size of the mob could scale according to the difficulty of the quest.  Certain named mobs could also have henchmen that defend them.

[Suggested](https://minecraft.curseforge.com/projects/baileys-dailies?comment=44) by snabul on curseforge

**Comment by frodare** [comment]
This code might be a useful reference:
https://github.com/ToroCraft/ToroQuest/blob/master/java/net/torocraft/toroquest/civilization/quests/QuestEnemyEncampment.java#L129

One of the problems I had was choosing mobs that can be in the sun light.

**Comment by pandapaul** [comment]
Nice!  That will absolutely be helpful.

**Comment by Twisted-Code** [comment]
what about spawning it in a cave? It might be a little mean, but it's not like people can't see the name tag through walls. Maybe only do that for the most difficult of quests, though. This should probably disqualify the mob from having any guards, but it maybe could still be a difficult type of mob, like nether?

**Comment by pandapaul** [comment]
I dig that and I didn't realize you can see name tags through walls.  That would make it viable I think.

**Comment by Twisted-Code** [comment]
yeah nametags show up just like player names do for players that aren't sneaking. That said, you may want to make the name **bold** if you're going to intentionally spawn it underground... or not if you want to be really mean/difficult 😛
